pallets/dispenser/src/extensions.rs

```diff
@@ -17,11 +17,7 @@
 // If you feel like getting in touch with us, you can do so at info@polimec.org
 
 use crate::{Call, Config};
-use frame_support::{
-	dispatch::{CheckIfFeeless, DispatchInfo},
-	pallet_prelude::*,
-	traits::{IsSubType, OriginTrait},
-};
+use frame_support::{dispatch::DispatchInfo, pallet_prelude::*, traits::IsSubType};
```

Comment:
We can also get rid of:

+ warning: unused import: `StaticTypeInfo`
+ warning: unused import: `PostDispatchInfoOf`